### PR TITLE
CO-717: User Delete flink job deployment script

### PR DESCRIPTION
**Pull request by amitpriyadarshi**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: ansible/inventory/env/group_vars/all.yml [hunk 1]
```diff
@@ -5,6 +5,8 @@ ansible_vault_password: "{{lp_vault_ansible_vault_password}}"
 env_name: "{{ env }}"
 instance_name: "{{ env }}"
 instance: "{{ env }}"
+azure_account: "{{ cloud_public_storage_accountname }}"
```
Comment (vinukumar-vs) [inline ansible/inventory/env/group_vars/all.yml]: Why do you need `azure_account` and `secret` for delete user functionality.

File: async-user-delete/pom.xml [hunk 2]
```diff
@@ -9,10 +9,10 @@
         <artifactId>cokreat-jobs</artifactId>
         <version>1.0</version>
     </parent>
-    <artifactId>user-delete</artifactId>
+    <artifactId>async-user-delete</artifactId>
     <version>1.0.0</version>
     <packaging>jar</packaging>
-    <name>user-delete</name>
+    <name>async-user-delete</name>
```
Comment (vinukumar-vs) [inline async-user-delete/pom.xml]: Why the name is having "async"?

File: async-user-delete/src/main/scala/org/sunbird/job/userdelete/domain/Event.scala [hunk 3]
```diff
@@ -7,26 +7,10 @@ class Event(eventMap: java.util.Map[String, Any], partition: Int, offset: Long)
 
 	private val jobName = "user-delete"
```
Comment (vinukumar-vs) [inline async-user-delete/src/main/scala/org/sunbird/job/userdelete/domain/Event.scala]: Why are we using the string "user-delete" again here? Its was changes to "async-user-delete" right

File: kubernetes/helm_charts/datapipeline_jobs/values.j2 [hunk 4]
```diff
@@ -1,10 +1,10 @@
 namespace: {{ flink_namespace }}
 imagepullsecrets: {{ imagepullsecrets }}
 dockerhub: {{ dockerhub }}
-repository: {{flink_repository|default('data-pipeline')}}
+repository: {{flink_repository|default('cokreat-jobs')}}
 image_tag: {{ image_tag }}
-azure_account: {{ cloud_public_storage_accountname }}
-azure_secret: {{ cloud_public_storage_secret }}
+azure_account: {{ azure_account }}
```
Comment (vinukumar-vs) [inline kubernetes/helm_charts/datapipeline_jobs/values.j2]: Code should be cloud agnostic right. We should not have any key names with CSP provider name

File: pom.xml [hunk 5]
```diff
@@ -20,9 +20,7 @@
 	<url>https://cokreat.sunbird.org/</url>
 	<modules>
 		<module>jobs-core</module>
-		<module>auto-creator-v2</module>
-		<module>content-auto-creator</module>
-		<module>user-delete</module>
+		<module>async-user-delete</module>
```
Comment (vinukumar-vs) [inline pom.xml]: Name "async-user-delete" doesn't looks good. 

Better you define with the prefix of "co-user-delete"(co-CoKreat) or just define as "user-delete"

**Review by vinukumar-vs — CHANGES_REQUESTED** [review]

File: ansible/inventory/env/group_vars/all.yml (hunk 1 above)
Comment (amitpriyadarshi) [reply ansible/inventory/env/group_vars/all.yml]: @vinukumar-vs I checked these configuration and getting used in jobs code and deployment scripts.. Therefor I have not removed these configuration.

File: async-user-delete/pom.xml (hunk 2 above)
Comment (amitpriyadarshi) [reply async-user-delete/pom.xml]: updated the name - as discussed

File: async-user-delete/src/main/scala/org/sunbird/job/userdelete/domain/Event.scala (hunk 3 above)
Comment (amitpriyadarshi) [reply async-user-delete/src/main/scala/org/sunbird/job/userdelete/domain/Event.scala]: updated

File: pom.xml (hunk 5 above)
Comment (amitpriyadarshi) [reply pom.xml]: updated the name - as discussed

File: kubernetes/helm_charts/datapipeline_jobs/values.j2 (hunk 4 above)
Comment (amitpriyadarshi) [reply kubernetes/helm_charts/datapipeline_jobs/values.j2]: did the req changes.

**Pull request merged**